apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -87,16 +87,17 @@ class ProgressBubbleSet extends React.Component {
       return true;
     }
     return false;
-  };
+  }
 
-  renderBubble = (level, index, isSublevel) => {
+  renderBubble(level, index, isSublevel) {
     const {
       levels,
+      studentProgress,
       selectedSectionId,
       selectedStudentId,
       hideAssessmentIcon
     } = this.props;
-
+    const levelProgress = studentProgress[level.id];
```

Comment:
Here, it looks like studentProgress is an array, but above, it's an object. Was that intentional?

Reply:
Copying an offline conversation here: 
PropTypes.objectOf means that all the values of the object conform to the type specified so that’s saying that studentProgress is an object with any type of key (in this case, level.id), and with values of type studentLevelProgressType

Reply:
✔️